crossdock/client/timeout/behavior.go

```diff
@@ -48,15 +55,49 @@ func Run(t crossdock.T) {
 
 	ch := raw.New(dispatcher.Channel("yarpc-test"))
 	_, _, err := ch.Call(yarpc.NewReqMeta(ctx).Procedure("sleep/raw"), nil)
-	fatals.Error(err, "expected a failure for timeout")
+	fatals.Error(err, "expected an error")
 
 	if transport.IsBadRequestError(err) {
 		t.Skipf("sleep/raw method not implemented: %v", err)
 		return
 	}
 
+	assert.True(transport.IsTimeoutError(err),
+		"expected a TimeoutError, got %T", err)
```

Comment:
For crossdock, `assert` will print this message in the output for both, success and failure, so you want it to be neutral. Something like,

    assert.True(transport.IsTimeoutError(err), "returns a TimeoutError")